test/node/__preperation.js

```diff
@@ -0,0 +1,15 @@
+const Circuit = require('../../circuit-node');
+const config = require('./config.json');
+let client = new Circuit.Client(config.bot1);
+let user;
+// client logs on to create a global conversation for other tests to use
+before(async () => {
+    try {
+        user = await client.logon();
+        global.conversation = await client.createConferenceBridge(`globalTestConversation${Date.now()}`);
```

Comment:
Rather than exposing it on "global" I though we could just make "preparation" a module, so any test that requires the global preparation would include it like this:

const { conversation } = require('./preparation');

Reply:
I think I got it to work but I'm not sure if how I did it is the best way.